### PR TITLE
Update reply-url.md

### PR DESCRIPTION
**Pull request by jesp209i**
Add that IDN is not supported for redirect urls

### CODE REVIEW

**Comment by Court72** [comment]
@henrymbuguakiarie

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by henrymbuguakiarie — COMMENTED** [review]
Hello @jesp209i,

Thank you for your contributions. Could you please explain why we need to add the sentence that "Redirect URIs don't support Internationalized Domain Names"? Also, have you tested every combination of Internationalized Domain Names (IDNs) to confirm that they don’t work?

**Comment by jesp209i** [comment]
hi @henrymbuguakiarie
I would not be able to test every combination of IDNs.

When trying to add a redirect url formed as an IDN the validation is always invalid an doesn't let me add the IDN. Also tried the punycoded variant of same hostname.

I have found pretty old entries elsewhere (old microsoft forum) explaining that this is not supported - but I feel like we are lacking official documentation for this.
https://serverfault.com/questions/985211/azure-app-registration-not-allowing-idn
https://feedback.azure.com/d365community/idea/607c6eaf-c625-ec11-b6e6-000d3a4f0789

**Comment by henrymbuguakiarie** [comment]
> @henrymbuguakiarie
> 
> Can you review the proposed changes?
> 
> **Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge.
> 
> #label:"aq-pr-triaged" @MicrosoftDocs/public-repo-pr-review-team

@jesp209i After doing some further digging and testing on IDNs, I agree with you that we should add that line. I’ll go ahead and merge the PR.

**Review by henrymbuguakiarie — APPROVED** [review]

**Comment by henrymbuguakiarie** [comment]
#sign-off

File: docs/identity-platform/reply-url.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ The Microsoft Entra application model specifies the following restrictions to re
     * `https://contoso.com/abc/response-oidc` is returned as `https://contoso.com/abc/response-oidc`
 
 * Redirect URIs *don't* support special characters - `! $ ' ( ) , ;`
+* Redirect URIs *doesn't* support Internationalized Domain Names
```
Comment (Court72) [inline docs/identity-platform/reply-url.md:109]:
```suggestion
* Redirect URIs *don't* support Internationalized Domain Names
```

**Pull request merged**